examples/client/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "grcp_client"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+hex = "0.4.3"
+pallas = "0.32.0"
```

Comment:
you shouldn't need Pallas here. You should be able to use it just with [utxorpc sdk](https://crates.io/crates/utxorpc)